### ISSUE TITLE
Link to Development docs is wrong

### ISSUE THREAD

**Issue by eamanu** [celery/celeryproject#27]
The link to development documentation is wrong on

![image](https://user-images.githubusercontent.com/7605307/56229792-1a37e900-6051-11e9-80e6-0af37e7c4855.png)

This redirect to: http://celery.github.io/celery/index.html and that response with ```404  code```

**Comment by cyberaa** [comment]
Still happening...

**Comment by eamanu** [comment]
yes, must be this link? https://docs.celeryproject.org/en/stable/
